docs/zkapps/tutorials/05-common-types-and-functions.mdx

```diff
@@ -59,34 +67,42 @@ Each type has the usual programming language semantics.
 
 For example, the following code:
 
-```ts
-const num1 = UInt32.from(40);
-const num2 = UInt64.from(40);
-
-const num1EqualsNum2: Bool = num1.toUInt64().equals(num2);
-
-console.log(`num1 === num2: ${num1EqualsNum2.toString()}`);
-console.log(`Fields in num1: ${num1.toFields().length}`);
-
-// --------------------------------------
-
-const signedNum1 = Int64.from(-3);
-const signedNum2 = Int64.from(45);
-
-const signedNumSum = signedNum1.add(signedNum2);
-
-console.log(`signedNum1 + signedNum2: ${signedNumSum}`);
-console.log(`Fields in signedNum1: ${signedNum1.toFields().length}`);
-
-// --------------------------------------
+<!--
 
-const char1 = Character.fromString('c');
-const char2 = Character.fromString('d');
-const char1EqualsChar2: Bool = char1.equals(char2);
-
-console.log(`char1: ${char1}`);
-console.log(`char1 === char2: ${char1EqualsChar2.toString()}`);
-console.log(`Fields in char1: ${char1.toFields().length}`);
+```ts src/run.ts
+1 import { UInt32, UInt64, Bool, Int64, Character, Signature, PrivateKey, CircuitString, Provable, Struct, Field, MerkleTree, MerkleWitness, MerkleMap,  Mina, SmartContract, AccountUpdate, method, state, State } from 'snarkyjs';
+2
+```
+-->
+
+```ts src/run.ts
+3 const num1 = UInt32.from(40);
+4 const num2 = UInt64.from(40);
```

Comment:
maintaining these line numbers going forward seems tedious to me :/

maybe there could be a default where it just appends to the file when there are no line numbers? (it'll still be tedious but at least only when the line numbers are needed)

Reply:
Yes, it is very tedious :( We would have to make some adjustments to the tutorial runner but possible (and probably worth it)

Reply:
YAY no more line number requirement! Let's omit the tedious line numbers per convo at offsite